cclib/parser/nwchemparser.py

```diff
@@ -59,6 +59,17 @@ def extract(self, inputfile, line):
                         "package_version"
                     ] = f"{self.metadata['package_version']}+{revision}"
 
+        if "nproc" in line:
+            self.metadata['processors'] = line.split()[-1]

```

Comment:
```suggestion
            self.metadata['num_processors'] = line.split()[-1]

```
otherwise it could be mistaken for processor names or types or ...